Examples/restcomm-olympus-nopod/restcomm-olympus/Utils.m

```diff
@@ -458,16 +431,43 @@ + (NSString*)convertInterappUri2RestcommUri:(NSURL*)uri
     return final;
 }
 
++ (NSArray *)getSortedContacts{
+    NSArray *contactsArray = [[NSArray alloc] init];
+    NSUserDefaults* appDefaults = [NSUserDefaults standardUserDefaults];
+    NSData *contactsArrayData = [appDefaults objectForKey:@"contacts"];
+    
+    if (contactsArrayData != nil) {
+        contactsArray = [NSKeyedUnarchiver unarchiveObjectWithData: contactsArrayData];
+        contactsArray = [Utils getNonDeletedFilteredContactArray:contactsArray];
+        if (contactsArray != nil){
+            //we dont want the default nubers to be sorted, so we will split an array and have them
+            //concatenate with the new array
+            NSArray *defaults = [Utils getDefaultFilteredContactArray:contactsArray forDefault:YES];
+            NSArray *others = [Utils getDefaultFilteredContactArray:contactsArray forDefault:NO];
+            
+         
+            NSSortDescriptor *sortFirstName = [NSSortDescriptor sortDescriptorWithKey:@"firstName" ascending:YES selector:@selector(caseInsensitiveCompare:)];
```

Comment:
Originally I was thinking of not separating demo with non demo when sorting as people, when having a sorted list, expect to not have exceptions to the sorting rule, but let's try it like this for now and see how it feels.